package.json

```diff
@@ -0,0 +1,87 @@
+{
+  "name": "UIProject",
+  "version": "0.0.1",
+  "private": true,
+  "dependencies": {
+    "@auth0/auth0-spa-js": "^2.1.2",
+    "@azure/msal-browser": "^3.5.0",
+    "@chakra-ui/icons": "^2.1.1",
+    "@chakra-ui/react": "^2.8.1",
+    "@emotion/react": "^11.11.1",
+    "@emotion/styled": "^11.11.0",
+    "@fontsource/open-sans": "^5.0.17",
+    "@fontsource/pt-sans": "^5.0.8",
+    "@fontsource/raleway": "^5.0.15",
+    "@reduxjs/toolkit": "^1.9.7",
+    "@types/react-copy-to-clipboard": "^5.0.6",
+    "@types/react-datepicker": "^4.10.0",
+    "@types/react-inner-image-zoom": "^3.0.1",
+    "@types/ramda": "^0.27.40",
+    "@types/react": "^18.2.31",
+    "@types/react-dom": "^18.2.14",
+    "@types/react-redux": "^7.1.28",
+    "@types/webpack-env": "^1.18.3",
+    "chakra-dayzed-datepicker": "^0.2.10",
+    "date-fns": "^2.30.0",
+    "dayzed": "^3.2.3",
+    "fp-ts": "^2.10.5",
+    "framer-motion": "^10.16.4",
+    "history": "^5.3.0",
+    "idtoken-verifier": "^2.2.3",
+    "io-ts": "^2.2.16",
+    "jszip": "^3.10.1",
+    "lodash": "^4.17.21",
+    "node-sass": "^8.0.0",
+    "prettier": "^2.8.3",
+    "ramda": "^0.29.0",
+    "react": "^18.2.0",
+    "react-copy-to-clipboard": "^5.1.0",
+    "react-date-picker": "^10.5.2",
+    "react-datepicker": "^4.21.0",
+    "react-dom": "^18.2.0",
+    "react-icons": "^4.11.0",
+    "react-inner-image-zoom": "^3.0.2",
+    "react-markdown": "^9.0.0",
+    "react-redux": "^8.1.3",
+    "react-router-dom": "^6.17.0",
+    "react-scripts": "5.0.1",
+    "react-tippy": "^1.4.0",
+    "redux": "^4.2.1",
+    "redux-observable": "^1.2.0",
+    "rxjs": "^6.6.7",
+    "typescript": "^5.2.2",
+    "web-vitals": "^2.1.4"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject"
+  },
+  "eslintConfig": {
+    "extends": [
+      "react-app",
+      "react-app/jest"
+    ]
+  },
+  "engines": {
+    "node": "19.9.0"
```

Comment:
bit strict a specific patch version. I'd pin to major only, like 19.* (not sure is the correct syntaxt!)

Reply:
actually 19 is not supported, so this should be either 18 or 20 (https://endoflife.date/nodejs)